cirq/protocols/qid_shape_protocol.py

```diff
@@ -17,11 +17,11 @@
 from typing_extensions import Protocol
 
 from cirq import ops
-from cirq._doc import document
+from cirq._doc import document, doc_private
 from cirq.type_workarounds import NotImplementedType
 
 if TYPE_CHECKING:
-    import cirq
+    pass
```

Comment:
Remove this block?

Reply:
Good catch - I thought I cleaned all those up - thanks!